src/components/app/components/Navbar.js

```diff
@@ -44,12 +42,9 @@ class Navbar extends React.Component {
     logout = ({ key }) => {
         const { isLoggedIn } = this.props;
 
-        if (isLoggedIn && key === SIGN_OUT) {
-            const { removeCookie, resetSigninState } = this.props.actions;
-
-            removeCookie();
-            message.info(VALID_SIGNOUT, 5);
-            resetSigninState();
+        if (isLoggedIn && key === SIGN_OUT ) {
```

Comment:
why would the key ever not be `SIGN_OUT` for the `logout` function. Is that a flow when it is not?

Reply:
yes, the key can be `USER_PROFILE`
This is it here
```<Menu onClick={this.logout}>
                <Item key={USER_PROFILE}>
                    <AppLink to={PROFILE} key={PROFILE}>{USER_PROFILE}</AppLink>
                </Item>
                <Item key={SIGN_OUT}>
                    <AppLink to={HOME} key={SIGN_OUT}>{SIGN_OUT}</AppLink>
                </Item>
            </Menu>
```